virtcontainers/types/sandbox.go

```diff
@@ -44,6 +46,16 @@ type State struct {
 
 	// GuestMemoryBlockSizeMB is the size of memory block of guestos
 	GuestMemoryBlockSizeMB uint32 `json:"guestMemoryBlockSize"`
+
+	// CgroupPath is the cgroup hierarchy where sandbox's processes
+	// including the hypervisor are placed.
+	CgroupPath string `json:"cgroupPath,omitempty"`
```

Comment:
This can bring potential broken compatibility.

Suppose sandboxA is created by old kata-runtime and runs for some time, after we upgrade kata-runtime, and when using the new kata-runtime to delete the old container, it will search `state.json` file from disk and end up to find nothing, The cgroup dir can fail to delete and garbage left.

That's why `https://github.com/kata-containers/runtime/pull/883` is important.

The influence is not that big caused by this modification, so I can accept the modification so far.